src/popup/ui.js

```diff
@@ -63,3 +78,73 @@ function createButtonsDiv(node, onReloadItems) {
 	buttons.appendChild(archiveButton)
 	return buttons
 }
+
+function loadLabelSelection(article, labels){
+	document.getElementById('content').style = 'display: none;'
```

Comment:
To make it consistent and not call `document.getElementById('content')` twice, you could put it in an extra variable here (and reuse that variable further down):

```suggestion
	const content = document.getElementById('content')
	content.style = 'display: none;'
```